features/support/publishing_api.rb

```diff
@@ -1,17 +1,14 @@
 require 'gds_api/publishing_api_v2'
 require_relative '../../test/support/policy_tagging_helpers'
 
-def stub_publishing_api_v1_requests
-  v1_url = Plek.current.find('publishing-api')
-  stub_request(:any, %r{\A#{ v1_url }})
-end
+PUBLISHING_API_V1_ENDPOINT = Plek.current.find('publishing-api')
```

Comment:
I feel like this is unnecessary indirection. `Plek.current.find` is a common pattern.